.tekton/release-service-pull-request.yaml

```diff
@@ -7,8 +7,9 @@ metadata:
     build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
     build.appstudio.redhat.com/target_branch: '{{target_branch}}'
     pipelinesascode.tekton.dev/max-keep-runs: "3"
-    pipelinesascode.tekton.dev/on-event: '[pull_request]'
-    pipelinesascode.tekton.dev/on-target-branch: '[main]'
+    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch
+      == "main" && ( "./***".pathChanged() || ".tekton/release-service-utils-pull-request.yaml".pathChanged()
```

Comment:
this doesn't seem right to me. Why is there a reference to utils in here? And why are we adding paths changed instead of just merges to main?

Reply:
good catch. must be a mistaken copy&paste on my part